openedx/tests/xblock_integration/test_done.py

```diff
@@ -0,0 +1,233 @@
+"""
+This tests that the completion XBlock correctly stores state. This
+is a fairly simple XBlock, and a correspondingly simple test suite.
+"""
+
+import json
+import mock
+import unittest
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+
+from lms.djangoapps.courseware.tests.helpers import LoginEnrollmentTestCase
+from lms.djangoapps.courseware.tests.factories import GlobalStaffFactory
+
+
+class SimpleEventTestMixin(object):
```

Comment:
We definitely want to move this into a location where it can be shared. Can you at least put it in a util file in openedx/tests/xblock_integration?

I see that it is similar to EventTestMixin in common/djangoapps/util/testing.py. Would it be possible to just use EventTestMixin, supplying your own tracker? Or is there not enough shared code to make it worth it?